src/inc/quic_datapath.h

```diff
@@ -222,6 +222,11 @@ typedef struct CXPLAT_RECV_DATA {
     //
     uint8_t TypeOfService;
 
+    //
+    // TTL Hoplimit field of the IP header of the received packet on handshake.
+    //
+    uint8_t HopLimitTTL;
```

Comment:
We should add a test in `DataPathTest.cpp` verifying packets received have non-zero values of this.

Reply:
https://github.com/microsoft/msquic/pull/4602/commits/4ee3e03b29c83cc6418970566758d86632ae053b#diff-548ffedbc89d78947ec587388ebe4a695dbaaf6aa8d9a5032f355eec08b98800